spec/requests/atmosphere/api/user_keys_spec.rb

```diff
@@ -184,12 +181,8 @@
         }
         post api("/user_keys", user), invalid_user_key_req
         expect(response.status).to eq 422
-        expect(json_response['details']).to eq({
-          'public_key' => [
-            'bad type of key (only ssh-rsa is allowed)',
-            'is invalid'
-          ]
-        })
+        expect(json_response['details']['public_key']).
+          to include('bad type of key (only ssh-rsa is allowed)')
```

Comment:
As far as I understand this spec (you are the original author :smile:) we care only about concrete error here.